.github/workflows/test-deploy.yml

```diff
@@ -67,7 +67,6 @@ jobs:
           cml runner --version
           cml publish --version
           cml pr --version
-          cml-pr --version
   packages:
```

Comment:
```suggestion
      - if: matrix.system != 'windows'
        run: |
          cml-runner --version
          cml-publish --version
          cml-pr --version
  packages:
```

Reply:
_Cough, cough._ Then, what do we want the `windows-latest` matrix platform?

Reply:
Note: we're already testing legacy commands with unit tests

https://github.com/iterative/cml/blob/7d664aec00023e8fba48e660f2f8e184477d8cdd/bin/legacy.test.js#L8